subworkflows/nf-core/bam_ngscheckmate/main.nf

```diff
@@ -0,0 +1,49 @@
+include { BCFTOOLS_MPILEUP } from '../../../modules/nf-core/bcftools/mpileup/main'
+include { NGSCHECKMATE_NCM } from '../../../modules/nf-core/ngscheckmate/ncm/main'
+
+workflow BAM_NGSCHECKMATE {
+
+    take:
+    ch_input            // channel: [ val(meta1), bam/cram ]
+    ch_snp_bed          // channel: [ val(meta2), bed ]
+    ch_fasta            // channel: [ val(meta3), fasta ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    ch_input_bed = ch_input.combine(ch_snp_bed.collect())
+                        // do something to combine the metas?
+                        .map{ input_meta, input_file, bed_meta, bed_file ->
+                            [input_meta, input_file, bed_file]
+                        }
+
+    BCFTOOLS_MPILEUP (ch_input_bed, ch_fasta, false)
+    ch_versions = ch_versions.mix(BCFTOOLS_MPILEUP.out.versions)
+
+    BCFTOOLS_MPILEUP
+    .out
+    .vcf
+    .map{meta, vcf -> vcf}    // discard individual metas
+    .collect()                // group into one channel
+    .map{files -> [files]}    // make the channel into [vcf1, vcf2, ...]
+    .set {ch_collected_vcfs}
```

Comment:
I thinkwe said we won't use set in sarek because of readability, but I am happy to merge this now as is so it can be in the release

Reply:
This is in a nf-core subworkflow, didn't realise `set` was a problem?

Reply:
ah sorry,, yeah never mind it was more a sarek wide thing.